packages/react/src/search-input/SearchInput.tsx

```diff
@@ -30,6 +30,7 @@ export const SearchInput = forwardRef<HTMLInputElement, SearchProps>(
           value && (
             <Button
               appearance="subtle"
+              aria-label="clear"
```

Comment:
let's use capital case here
```suggestion
              aria-label="Clear"
```

also i see that this button isn't focusable in native search box of browser - so we should do the same

let's add a `tabIndex={-1}` to this button